server/Knowledge_checklist.sql

```diff
@@ -0,0 +1,100 @@
+drop table if exists students cascade;
+drop table if exists mentors cascade;
+drop table if exists region;
+drop table if exists learning_objectives cascade;
+drop table if exists mapping_skills cascade;
+drop table if exists tech_skills cascade;
+drop table if exists competency_level cascade;
+
+CREATE TABLE students (
+    id          SERIAL PRIMARY KEY,
+    name        VARCHAR (50)NOT NULL,
+    location    VARCHAR (50)NOT NULL,
+    email       VARCHAR (150) NOT NULL,
+    password    VARCHAR(70) NOT NULL
+);
+
+CREATE TABLE mentors (
+    id          SERIAL PRIMARY KEY,
+    name        VARCHAR (50)NOT NULL,
+    location    VARCHAR (50)NOT NULL,
+    email       VARCHAR (150) NOT NULL,
+    password    VARCHAR(70) NOT NULL
+);
+
+CREATE TABLE region (
+    id          SERIAL PRIMARY KEY,
+    location    VARCHAR(50) NOT NULL 
+);
+
+CREATE TABLE learning_objectives (
+    id           SERIAL PRIMARY KEY,
+    obj_id       VARCHAR (50)NOT NULL,
+    lesson_id    INT NOT NULL            
+);
+
+CREATE TABLE mapping_skills (
+    id          SERIAL PRIMARY KEY,
+    stud_id     INT REFERENCES students(id), 
+    obj_id      INT NOT NULL,    
```

Comment:
I assume that `obj_id` and `comp_id` are also foreign keys to other tables so they could also have the `REFERENCES` directive.

Reply:
AO amended